frontend/src/components/common/Header/Profile/Desktop.svelte

```diff
@@ -45,7 +45,6 @@
           />
         </button>
       </div>
-
```

Comment:
What do you think about module and logical separation between block or module?

Reply:
I dont understand what you do want...
I prefer discuss that in discord

> What do you think about module and logical separation between block or module?